### PR TITLE
Yinon

### PR DESCRIPTION
**Pull request by Yinonss**
Add oscillator and heartbeat components.

### CODE REVIEW

File: Assets/Scripts/Oscillator.cs [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    float speed = 5f;
+    [SerializeField]
+    float acceleration = -1f;
+    float mid;
+    // Start is called before the first frame update
+
+
+    void Start()
+    {
+        Debug.Log("Start");
```
Comment (DYMGames) [inline Assets/Scripts/Oscillator.cs]: delete it

File: Assets/Scripts/ScaleChanger.cs [hunk 2]
```diff
@@ -0,0 +1,29 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+/**
+ * This script expands and shrinks object size in a heartbeat motion.
+ * */
+public class ScaleChanger : MonoBehaviour
+{
+	private float d = 1f;
```
Comment (DYMGames) [inline Assets/Scripts/ScaleChanger.cs]: tab fix

File: Assets/Scripts/ScaleChanger.cs [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    // Start is called before the first frame update
+    void Start()
+    {
+
+    }
+    // Update is called once per frame
+    void Update()
+    {
+        temp = transform.localScale;
+        transform.localScale +=temp *d * Time.deltaTime;
```
Comment (DYMGames) [inline Assets/Scripts/ScaleChanger.cs]: transform.localScale += temp * d  * Time.deltaTime;

**Review by DYMGames — CHANGES_REQUESTED** [review]

File: Assets/Scripts/Oscillator.cs (hunk 1 above)
Comment (maor6) [reply Assets/Scripts/Oscillator.cs]: not relevant

File: Assets/Scripts/ScaleChanger.cs (hunk 2 above)
Comment (maor6) [reply Assets/Scripts/ScaleChanger.cs]: you can delete the private

**Pull request merged**